internal/supervisor/installation.go

```diff
@@ -953,44 +953,6 @@ func (s *InstallationSupervisor) waitForUpdateStable(installation *model.Install
 	return model.InstallationStateStable
 }
 
-// Unused stub function
```

Comment:
C'mon linter! It says right there it's unused. Why do you just leave it alone!? 😂